renovate.json

```
@@ -1,6 +1,5 @@
 {
   "extends": [
-    "eslint-config-unjs"
-  ],
-  "rules": {}
+    "@nuxtjs"
```

Thanks for spotting this 👌 

```suggestion
    "github>unjs/renovate-config"
```